src/wovnio/wovnphp/Store.php

```diff
@@ -47,7 +47,7 @@ private function defaultSettings() {
         'url_pattern_name' => 'query',
         'url_pattern_reg' => '((\?.*&)|\?)wovn=(?P<lang>[^&]+)(&|$)',
         'query' => array(),
-        'api_url' => 'https://api.wovn.io/v0/',
+        'api_url' => 'http://wovn.global.ssl.fastly.net/v0/',
```

Comment:
Looks good.
Why don't you use `https` instead of `http`?

Reply:
done